pkg/cmd/repo_list.go

```diff
@@ -40,7 +39,8 @@ func newRepoListCmd(out io.Writer) *cobra.Command {
 		RunE: func(_ *cobra.Command, _ []string) error {
 			f, _ := repo.LoadFile(settings.RepositoryConfig)
 			if len(f.Repositories) == 0 && outfmt != output.JSON && outfmt != output.YAML {
-				return errors.New("no repositories to show")
+				fmt.Fprintln(out, "no repositories to show")
```

Comment:
I would think this would go to stderr so it isn't interpreted as a repo

Reply:
thanks for review, I you mean that it might go to stdout and break the JSON pipeline, just did a change let me know if there are more issues

Reply:
That makes sense to me. We'll see what @gjenkins8 thinks